doc/README.md

```diff
@@ -40,19 +41,30 @@ Usage example: shut down entire branch of related products or "release went wron
 
 Another built-in nice to have feature is Spring authorities.
 At the moment it is poorly implemented and lack of proper management if there is any management at all.
-There are only 3 roles present which are used for accessing API and web UI itself:
+Spring security is configured for both: API and web UI.
+In case BasicAuth is provided when **GET**ting feature check it'll take into account built-in roles:
```

Comment:
**GET**ting ? Not sure if it is intentional ?

Reply:
Instead of BasicAuth can we say basic authentication ?

Reply:
Sorry, was intentional. Pretty much tech-y. Afterall, you need some tech knowledge in order to understand/use this